docs/root/configuration/http/http_filters/cache_v2_filter.rst

```diff
@@ -63,21 +72,39 @@ Example filter configuration with a ``SimpleHttpCache`` cache implementation:
    :lineno-start: 29
    :caption: :download:`http-cache-v2-configuration.yaml <_include/http-cache-v2-configuration.yaml>`
 
+Example filter configuration with a ``FileSystemHttpCacheV2`` cache implementation:
+
+.. literalinclude:: _include/http-cache-v2-configuration-fs.yaml
+   :language: yaml
+   :start-at: http_filters:
+   :end-before: envoy.filters.http.router
+   :linenos:
+   :lineno-match:
+   :caption: :download:`http-cache-v2-configuration-fs.yaml <_include/http-cache-v2-configuration-fs.yaml>`
+
+
 The more complicated filter chain configuration required if mutations occur upstream of the cache filter
 involves duplicating the full route config into an internal listener (unfortunately this is currently unavoidable):
 
 .. literalinclude:: _include/http-cache-v2-configuration-internal-listener.yaml
    :language: yaml
-   :lines: 38-113
+   :start-at: http_filters:
    :linenos:
-   :lineno-start: 38
+   :lineno-match:
    :caption: :download:`http-cache-v2-configuration-internal-listener.yaml <_include/http-cache-v2-configuration-internal-listener.yaml>`
 
 .. TODO(ravenblackx): Add sandbox and link it below, similar to what cache_filter does.
 
-.. TODO(ravenblackx): Update the docs like the recent update to the old cache docs.
-
 .. seealso::
 
+   :ref:`HTTP CacheV2 filter (proto file) <envoy_v3_api_file_envoy/extensions/filters/http/cache_v2/v3/cache.proto>`
```

Comment:
lets reference the original cache filter even if its with "now deprecated" or somesuch - i think we need to actively deconfuse this

same other way - i think we need to add a seealso there

Reply:
That reference is here from the previous PR (just after this change-block), and so is the one the other way.

Reply:
Oh, you're right, that seems like a mistake. Should link between these pages and not to the proto.

Reply:
i think both - we defo want the proto link - but we also want to acknowledge that there is a different cache impl - ie different config etc